iam/policy-templates/ci-cd.json

```diff
@@ -125,6 +125,9 @@
         "dynamodb:ListTables",
         "dynamodb:Query",
         "dynamodb:DescribeTable",
+        "dynamodb:GetItem",
+        "dynamodb:PutItem",
+        "dynamodb:DeleteItem",
         "sqs:ListQueues"
 	],
       "Resource": "*"
```

Comment:
might be good to specify the resource, instead of all dynamodb resources; principal of least access?

Reply:
I was considering this too, though more to conserve space since the travis file has gotten too long (there's a size limit) once before.  I'll look and see.

Reply:
Changed.